docs/running_rune_with_occlum.md

```diff
@@ -4,26 +4,42 @@
 ## Requirements
 - Install [Intel SGX driver for Linux](https://github.com/intel/linux-sgx-driver#build-and-install-the-intelr-sgx-driver), required by Intel SGX SDK && PSW.
 - Install [enable_rdfsbase kernel module](https://github.com/occlum/enable_rdfsbase#how-to-build), allowing to use `rdfsbase` -family instructions in Occlum.
-- Assume the host system is CentOS 7.5.
-- You can also launch a CentOS 7.5 container as your host system with the following command:
-  ```shell
-  docker run -it --privileged --device /dev/isgx centos:7.5.1804
-  ```
-  If so, you need to run **another docker daemon** inside this CentOS 7.5 container. Please refer to [this guide](https://docs.docker.com/engine/install/centos) to install docker daemon, and type the following command to start dockerd.
-  ```shell
-  dockerd -b docker0 --storage-driver=vfs &
-  ```
+- Ensure that you have one of the following required operating systems:
+	- CentOS 7.5. You can also launch a CentOS 7.5 container as your host system with the following command:
+	```shell
+	docker run -it --privileged --device /dev/isgx centos:7.5.1804
+	```
+	If so, you need to run **another docker daemon** inside this CentOS 7.5 container. Please refer to [this guide](https://docs.docker.com/engine/install/centos) to install docker daemon, and type the following command to start dockerd.
+  	```shell
+  	dockerd -b docker0 --storage-driver=vfs &
+  	```
+	- Ubuntu 18.04. You can also launch a Ubuntu 18.0 container as your host system with the following command:
+  	```shell
+  	docker run -it --privileged --device /dev/isgx ubuntu:18.04
+  	```
+  	If so, you need to run **another docker daemon** inside this Ubuntu 18.04 container. Please refer to [this guide](https://docs.docker.com/engine/install/ubuntu) to install docker daemon, and type the following command to start dockerd.
+  	```shell
+  	service docker start
+  	```
 
 ---
 
 ## Build Occlum application container image
 ### Download Occlum SDK container image
+- On CentOS 7.5: 
 ```shell
 mkdir "$HOME/rune_workdir"
 docker run -it --privileged --device /dev/isgx \
   -v "$HOME/rune_workdir":/root/rune_workdir \
   occlum/occlum:0.14.0-centos7.5
 ```
+- On Ubuntu 18.04:
+```shell
+mkdir "$HOME/rune_workdir"
+docker run -it --privileged --device /dev/isgx \
+  -v "$HOME/rune_workdir":/root/rune_workdir \
+  occlum/occlum:0.14.0-ubuntu18.04
+```
```

Comment:
hmm... The problem here is: do we need to describe two kinds of occlum here? Note that any occlum sdk is container not a host. Successfully launching one kind of occlum sdk container is not enough? Are we educating the user how to launch different occlum sdk container?